src/Microsoft.Diagnostics.Monitoring.Egress/Microsoft.Diagnostics.Monitoring.Egress.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Given how small the Egress api + default implementation are, I think it would be ok for them to be in the base Monitoring dll. I think Egress.AzureStorage should still be its own assembly.

Reply:
I merged the egress primitives into the Monitoring assembly.